components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -311,7 +312,14 @@ class AddonsManagerAdapter(
         // Add recommended section and addons if available
         if (recommendedAddons.isNotEmpty()) {
             itemsWithSections.add(Section(R.string.mozac_feature_addons_recommended_section, true))
-            itemsWithSections.addAll(recommendedAddons)
+            if (excludedAddons != null && excludedAddons.isNotEmpty()) {
```

Comment:
If we made the list a non null we could avoid this check `excludedAddons != null`.